packages/xo-server-backup-reports/src/index.js

```diff
@@ -124,12 +126,124 @@ const addWarnings = (text, warnings, nbIndent = 0) => {
     return
   }
 
-  const indent = INDENT.repeat(nbIndent)
+  const indent = TAB.repeat(nbIndent)
   warnings.forEach(({ message }) => {
     text.push(`${indent}- **${ICON_WARNING} ${message}**`)
   })
 }
 
+// ===================================================================
+
+const TITLE_BY_STATUS = {
+  failure: n => `## ${n} Failure${n === 1 ? '' : 's'}`,
+  interrupted: n => `## ${n} Interrupted`,
+  skipped: n => `## ${n} Skipped`,
+  success: n => `## ${n} Success${n === 1 ? '' : 'es'}`,
+}
+
+const getTemporalDataMarkdown = (end, start, formatDate) => {
+  const markdown = [`- **Start time**: ${formatDate(start)}`]
+  if (end !== undefined) {
+    markdown.push(`- **End time**: ${formatDate(end)}`)
+    const duration = end - start
+    if (duration >= 1) {
+      markdown.push(`- **Duration**: ${formatDuration(duration)}`)
+    }
+  }
+  return markdown
+}
+
+const getWarningsMarkdown = warnings =>
+  warnings.map(({ message }) => `- **${ICON_WARNING} ${message}**`)
+
+const getError = task => {
+  let message
+  if (
+    task.status === 'success' ||
+    (message = defined(() => task.result.message, () => task.result.code)) ===
+      undefined
+  ) {
+    return
+  }
+
+  const label = task.status === 'skipped' ? 'Reason' : 'Error'
+  return `- **${label}**: ${message}`
+}
+
+const poolMarkdown = (task, { formatDate }) => {
```

Comment:
Why do you sometime prefix with `get` but not always?

Does it mean something?

Reply:
Forgotten.